.circleci/config.yml

```diff
@@ -0,0 +1,79 @@
+version: 2
+references:
+  base: &base
+    docker:
+      - image: circleci/ruby:2.6.3
+        environment:
+          RAILS_ENV: test
+          PGHOST: 127.0.0.1
+          PGUSER: bitcrowd
+          DATABASE_URL: "postgres://bitcrowd@localhost:5432/bitcrowd_test"
+          COVERAGE: 'true'
+          BUNDLE_PATH: '~/bitcrowd/vendor/bundle'
+          DISPLAY: 99.0
+          CAPYBARA_WAIT_TIME: 40
+      - image: circleci/postgres:9.6.12-alpine
+        environment:
+          POSTGRES_USER: bitcrowd
+          POSTGRES_DB: bitcrowd_test
+    working_directory: ~/bitcrowd
+jobs:
+  setup:
+    <<: *base
+    steps:
+      - checkout
+
+      # ruby dependencies
+      - restore_cache:
+          keys:
+            - bundler-cache-v1-{{ checksum "Gemfile.lock" }}
+            - bundler-cache-v1-
+      - run: gem install bundler
+      - run:
+          name: Install ruby dependencies
+          command: |
+            bundle check || bundle install --jobs 4 --retry 3
+            bundle clean
+      - save_cache:
+          key: bundler-cache-v1-{{ checksum "Gemfile.lock" }}
+          paths:
+            - vendor/bundle
+
+      - save_cache:
+          key: repo-{{ .Environment.CIRCLE_SHA1 }}
+          paths:
+            - ~/bitcrowd
+  test:
+    <<: *base
+    steps:
+      - restore_cache:
+          key: repo-{{ .Environment.CIRCLE_SHA1 }}
+      - run: 
+          name: Install postgresql-client
+          command: |
+            sudo apt-get update
+            sudo apt install -y postgresql-client
+      - run:
+          name: rubocop
+          command: |
+            bundle exec rubocop
+      - run:
+          name: Wait for DB
+          command: dockerize -wait tcp://localhost:5432 -timeout 1m
+      - run:
+          name: Database Setup
+          command: |
+            bundle exec rails db:drop || true
```

Comment:
Same here: I don't think it's necessary to drop the database.

Reply:
good spoeeted 👍  fixed here and in that other PR